docs/src/main/tut/resources_for_learners.md

```diff
@@ -17,3 +17,13 @@ position: 3
    offers multiple online exercises to learn about the type classes and 
    datatypes included in Cats.
    
+# Videos
+ * [Functional Structures in Scala](https://www.youtube.com/playlist?list=PLFrwDVdSrYE6dy14XCmUtRAJuhCxuzJp0) 
+   by [Michael Pilquist](https://github.com/MPilquist) offers a series
+   of lectures about basic functional structures. 
+ * [Category Theory for Programmers](https://www.youtube.com/playlist?list=PLbgaMIhjbmEnaH_LTkxLI7FMa2HsnawM_) 
+   by [Bartosz Milewski](https://github.com/BartoszMilewski) is a wonder
```

Comment:
Sorry, 2 kind of nitpicky questions/comments.

What exactly do you mean by "wonder series"? Did you mean "wonderful series"? If so, "wonderful" is probably fine, but it might be better to use a more objective description.

I'm a bit hesitant to say "should be easy for non-Haskell programmers to follow". Even introductory category theory and haskell can be a big paradigm shift for people, and I don't want to discourage people or have them feel bad about themselves if they don't easily follow it.